test/cmd/projects.sh

```diff
@@ -15,13 +15,18 @@ os::cmd::expect_failure_and_text 'oc projects test_arg' 'no arguments'
 os::cmd::expect_success 'oc login -u test -p test'
 os::cmd::expect_success_and_text 'oc projects' 'You are not a member of any projects'
 # add a project and expect text for a single project
-os::cmd::expect_success_and_text 'oc new-project test4; sleep 2; oc projects' 'You have one project on this server: "test4".'
-os::cmd::expect_success_and_text 'oc new-project test5; sleep 2; oc projects' 'You have access'
+os::cmd::expect_success_and_text 'oc new-project test4' 'Now using project "test4" on server '
+# avoid flakes in case of slow server update
```

Comment:
Polling instead of `sleep` is "my scripts are running on some remote server 101" so while I agree that it's at least somewhat useful information, if we put it every single time we made the decision to poll instead of wait we'd double the length of our tests